### DISCUSSION TITLE
Paste plain text shortcut

### DISCUSSION THREAD

**Discussion by r1y4h** [xdan/jodit#1091] [Q&A] [answered]
I most cases in Windows you can do "shift+ctrl+v" to initiate paste plain text to an editor. Jodit doesn't seem to support this paste plain text shortcut. We want to be able to distinguish pasting of formatted text (ctrl+v) vs plain text (shit+ctrl+v) without using any modal.

**Comment by JaydevKalariyaa** [comment]
did you able to find any solution for it?

**Comment by xdan** [comment, marked as answer]
Jodit doesn't ship a separate Shift+Ctrl+V shortcut — paste routing is controlled by `defaultActionOnPaste` (`insert_as_html`, `insert_clear_html` or `insert_only_text`) plus the ask-dialog, so there's no built-in "this hotkey = plain, that one = formatted" split.

You can wire it yourself: catch Shift+Ctrl+V in a keydown handler and run that paste through the plain-text path (or flip `defaultActionOnPaste` to `insert_only_text` for it) while plain Ctrl+V keeps `insert_as_html`. Tell me your setup and I'll show a minimal handler.
